### PR TITLE
#175 Characters born year 20 or earlier displays age "???"

### LINKED ISSUE #175: Custom Born Dates
**Describe your suggestion in full detail below:**
Try to tweak the interface and localization a bit to show that Tyrande and other really old characters were born a long time ago.
<details>
<summary>Click to expand</summary>

image

</details>
I advise using some kind of character variable alternative to the character's real age.

### PR DESCRIPTION
**Pull request by robmart**
Resolves #175 

## Changelog:
- Characters born year 20 or earlier displays age "???"

## Developer changelog:
- Edited the vanilla age text to only display if birth year is greater than 20
- Added new text box to display when birth year is lesser or equal to 20
    - The text is "???"
    - Custom popup

# How to test
    - Make sure no ages over 562 are shown (in the 582 startdate). Check all UI that shows age.

# TODO
- ~~Char select screen~~

### CODE REVIEW

File: localization/german/wc_tooltip_l_german.yml [hunk 1]
```diff
@@ -33,4 +33,14 @@
...
\ No newline at end of file
+ is_isolated_in_uldum_tooltip_first_not:0 "Der Schleier von Uldum erlaubt Interaktionen zwischen [CHARACTER.GetShortUIName|U] und [TARGET_CHARACTER.GetShortUIName|U]"
+  
+ shatter_realm_tooltip_first:0 "Your realm falls apart freeing [vassals|E] and giving lands to [claimants|E]"
+ shatter_realm_tooltip_first_past:0 "Your realm falled apart freeing [vassals|E] and giving lands to [claimants|E]"
+ shatter_realm_tooltip_global:0 "[CHARACTER.GetFirstNamePossessive|U] realm falls apart freeing [vassals|E] and giving lands to [claimants|E]"
+ shatter_realm_tooltip_global_past:0 "[CHARACTER.GetFirstNamePossessive|U] realm falled apart freeing [vassals|E] and giving lands to [claimants|E]"
+
+ leave_union_tooltip_global:0 "[CHARACTER.GetShortUIName|U] leaves the current [union|E]"
+ leave_union_tooltip_global_past:0 "[CHARACTER.GetShortUIName|U] left the current [union|E]"
```
Comment (zumbak04) [inline localization/german/wc_tooltip_l_german.yml:44]: Looks like it's from another PR. You should clear it.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: localization/german/wc_tooltip_l_german.yml (hunk 1 above)
Comment (robmart) [reply localization/german/wc_tooltip_l_german.yml:44]: @arithon

Comment (robmart) [reply localization/german/wc_tooltip_l_german.yml]: Fixed in 972ddd2

File: gui/window_character.gui [hunk 2]
```diff
@@ -726,7 +726,9 @@ window = {
 								tooltip = "[Character.GetUINameNotMeNoTooltip]"
```
Comment (zumbak04) [inline gui/window_character.gui:726]: You can find out the age in the character finder.
<details>
<summary>Click to expand</summary>



</details>

Comment (zumbak04) [inline gui/window_character.gui:726]: You can find out the age in the character finder.
<details>
<summary>Click to expand</summary>

image

</details>

Comment (robmart) [reply gui/window_character.gui:726]: > You can find out the age in the character finder.
> Click to expand

And also the Character select screen

File: localization/german/wc_tooltip_l_german.yml (hunk 1 above)
Comment (arithon) [reply localization/german/wc_tooltip_l_german.yml]: I intentionally added them so the file would be complete...

File: gui/window_character_finder.gui [hunk 3]
```diff
@@ -0,0 +1,178 @@
+### LIST TEMPLATE
```
Comment (zumbak04) [inline gui/window_character_finder.gui:1]: Add this file to the master and merge to this branch so I can see what you changed.

Comment (robmart) [reply gui/window_character_finder.gui]: Done in a3ce38f

File: gui/window_character_finder.gui [hunk 4]
```diff
@@ -72,6 +72,28 @@ window = {
...
+								#Warcraft
+								text_single	 = {
+									visible = "[LessThanOrEqualTo_int32(Character.GetBirthDate.GetYear, '(int32)20')]"
+									name = "character_age_2"
+									text = " ???"
+									default_format = "#low"
+									fontsize = 15
+									align = nobaseline
+								}
+							}
```
Comment (zumbak04) [inline gui/window_character_finder.gui:96]: I don't think you need this part if you changed `lists.gui`.

Comment (robmart) [reply gui/window_character_finder.gui:96]: > 
> 
> I don't think you need this part if you changed `lists.gui`.

That's probably true. Changed that part before I changed lists.gui

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by arithon — APPROVED** [review]
loc LGTM

**Review by hamjustine — APPROVED** [review]

**Pull request merged**